gradle/license.gradle

```diff
@@ -0,0 +1,22 @@
+apply plugin: 'com.github.hierynomus.license'
```

Comment:
why are we adding this file? 

project template does not have it https://github.com/micronaut-projects/micronaut-project-template/tree/master/gradle